contrib/scripts/codepage_to_utf8.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python3
+"""
+Convert file from legacy codepage to Unicode UTF-8
+"""
```

Comment:
Add an SPDX and copyright line to clarify the file's license. 

Use `scripts/count-pvs-bugs.py` as an example. 

Suggest a bit more explanation about the tool, and can mentioning that the primary use of this is to convert GOG conf files, as an example use case.

Reply:
Done, let me know if you think this is sufficient.